application-task-ui/src/main/resources/TaskManager/ConfluenceTasksReportMacroBridge.xml

```diff
@@ -0,0 +1,1050 @@
+        <classType>com.xpn.xwiki.objects.classes.TextAreaClass</classType>
+      </description>
+      <id>
+        <disabled>0</disabled>
+        <name>id</name>
+        <number>1</number>
+        <prettyName>Macro id</prettyName>
+        <size>30</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
+      </id>
+      <name>
+        <disabled>0</disabled>
+        <name>name</name>
+        <number>2</number>
+        <prettyName>Macro name</prettyName>
+        <size>30</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
+      </name>
+      <priority>
+        <disabled>0</disabled>
+        <name>priority</name>
+        <number>11</number>
+        <numberType>integer</numberType>
+        <prettyName>Priority</prettyName>
+        <size>10</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.NumberClass</classType>
+      </priority>
+      <supportsInlineMode>
+        <disabled>0</disabled>
+        <displayFormType>select</displayFormType>
+        <displayType>yesno</displayType>
+        <name>supportsInlineMode</name>
+        <number>5</number>
+        <prettyName>Supports inline mode</prettyName>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.BooleanClass</classType>
+      </supportsInlineMode>
+      <visibility>
+        <cache>0</cache>
+        <disabled>0</disabled>
+        <displayType>select</displayType>
+        <freeText>forbidden</freeText>
+        <largeStorage>0</largeStorage>
+        <multiSelect>0</multiSelect>
+        <name>visibility</name>
+        <number>6</number>
+        <prettyName>Macro visibility</prettyName>
+        <relationalStorage>0</relationalStorage>
+        <separator>|</separator>
+        <separators>|</separators>
+        <size>1</size>
+        <unmodifiable>0</unmodifiable>
+        <values>Current User|Current Wiki|Global</values>
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+      </visibility>
+    </class>
+    <property>
+      <async_cached>0</async_cached>
+    </property>
+    <property>
+      <async_context/>
+    </property>
+    <property>
+      <async_enabled>0</async_enabled>
+    </property>
+    <property>
+      <code>{{velocity}}
+#set ($pages = $xcontext.macro.params.spaces)
+#set ($tags = $wikimacro.parameters.labels)
+#set ($assignees = $xcontext.macro.params.assignees)
+#set ($creator = $xcontext.macro.params.creators)
+#set ($createdAfter = $wikimacro.parameters.createddateFrom)
+#set ($status = $xcontext.macro.params.status)
+#set ($sortBy = $wikimacro.parameters.sortBy)
+#set ($displayColumns = $wikimacro.parameters.columns)
+#set ($rowCount = $wikimacro.parameters.pageSize)
+#set ($paramMap = {})
+#if ("$!pages" != '')
+  #set ($discard = $paramMap.put('pages', "pages='$pages'"))
```

Comment:
Missing wiki syntax escaping here and for most of the entries you add to the map (to prevent wiki syntax injection). Also, I don't understand why you repeat the key in the value. I find it easier to put only the value here and to handle the wiki syntax generation and escaping at the end, in a single place.